sources/packages/backend/workflow/test/2022-2023/parttime-assessment/awards-amounts/parttime-assessment-awards-amount-BCAG.e2e-spec.ts

```diff
@@ -185,6 +185,42 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-awards-amount-BC
     expect(calculatedAssessment.variables.provincialAwardNetBCAGAmount).toBe(0);
   });
 
+  it(
+    "Should determine federalAwardBCAGAmount to be 1000, provincialAwardNetBCAGAmount to be 700 when " +
+      "student's current year income is 20001 and their relationship status is single.",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataCurrentYearIncome = 20001;
```

Comment:
Please include the `studentDataCRAReportedIncome` to ensure the `studentDataCurrentYearIncome` takes precedence.